src/Compilers/Core/Portable/PEWriter/MetadataWriter.PortablePdb.cs

```diff
@@ -990,5 +991,28 @@ private void EmbedMetadataReferenceInformation(CommonPEModuleBuilder module)
                 kind: _debugMetadataOpt.GetOrAddGuid(PortableCustomDebugInfoKinds.CompilationMetadataReferences),
                 value: _debugMetadataOpt.GetOrAddBlob(builder));
         }
+
+        private void TypeDocumentInformation(CommonPEModuleBuilder module)
+        {
+            DocumentHandle rowid;
```

Comment:
>DocumentHandle rowid; [](http://example.com/codeflow?start=12&length=21)

This declaration doesn't need to be here - you can declare rowid where it's used.